eqcorrscan/utils/clustering.py

```diff
@@ -119,6 +143,11 @@ def distance_matrix(stream_list, shift_len=0.0, cores=1):
         matrix for
     :type shift_len: float
     :param shift_len: How many seconds for templates to shift
```

Comment:
```suggestion
    :param shift_len:
        Seconds to shift the streams by (total value for negative and positive
        direction together)
```
To match above, which is clearer than my old version!